changelogs/unreleased/gh-10506-spam-in-netbox-logs.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/netbox
```

Comment:
Nit: should be smth like `lua`. You can check the allowed here:

https://github.com/tarantool/tarantool/blob/cc77a6ead00b9a07233f302dabae994fc79dde21/tools/gen-release-notes#L15-L16